docs/reference/aggregations/pipeline/inference-bucket-aggregation.asciidoc

```diff
@@ -53,8 +53,8 @@ See <<buckets-path-syntax>> for more details | Required       | -
 
 ==== Configuration options for {infer} models
 
-The `inference_config` setting is optional and usually isn't required as the 
-pre-trained models come equipped with sensible defaults. In the context of 
+The `inference_config` setting is optional and usually isn't required as the
```

Comment:
Is the ultimate plan to also rename the `inference_config` field?

Reply:
no. `inference_config` is the default configuration when using the model during inference.